patchwork-extensions-shearing/src/main/java/com/patchworkmc/impl/extension/shearing/Shearables.java

```diff
@@ -17,7 +17,7 @@
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
  */
 
-package com.patchworkmc.impl.extensions.shearing;
+package com.patchworkmc.impl.extension.shearing;
```

Comment:
This is an unrelated fix, this package name didn't match the ones used everywhere else.

Reply:
No? `com.patchworkmc.mixin.extensions.shearing` is what all of the shearing mixins use, and MinecraftForge uses `net.minecraftforge.common.extensions`. The only exception is MixinStatusEffect. If you want to rename all of the other patchwork packages, fine, but this is only halfway. I think this refactor makes sense for another PR. I'm still split if `extension` or `extensions` makes more sense for the package name: Forge uses `extensions`, and most of patchwork uses `extensions`.